### DISCUSSION TITLE
Licensing and Pricing for Highcharts

### DISCUSSION THREAD

**Discussion by brianhumphreys** [twilio-labs/paste#3412] [Q&A] [answered]
We are planning on using the Highcharts library to create a Dashboard for Segment Engage analytics.  We need to use Pie Charts, Line Charts and Bar Charts.  I understand that we already have licensing worked out for Highcharts usage in Twilio.  I am wondering if we need any additional licensing to utilize the library in the segment app.

[Dashboard PRD](https://docs.google.com/document/d/1I3mpwWUn0-sMuaddL7Zliet94T5c0wERktn7F0i31hU/edit#heading=h.epzvpsyrpu3j)
[Figma](https://www.figma.com/file/npHoqZDj3hLxMQ8dbY1eaH/%5BETP%5D-Marketing-Analytics?type=design&node-id=1990-353843&mode=design&t=QaVYsDjEqkSIIwwF-0)

**Comment by SiTaggart** [comment, marked as answer]
Hi @brianhumphreys 

Yes, Highcharts is licensed for use on Twilio applications, including Segment. The Data and Apps org even contributed to the cost when it went through procurement last year.

The architecture decision record was posted here https://github.com/twilio-internal/architecture/blob/main/language-guilds/typescript-guild/adr/0002-charting-library.md
